src/app/clients/client-stepper/client-general-step/client-general-step.component.html

```
@@ -146,8 +144,8 @@
     </mat-form-field>
 
     @if (createClientForm.get('clientNonPersonDetails')) {
-      <div class="flex-100 layout-row-wrap responsive-column gap-4percent" formGroupName="clientNonPersonDetails">
-        <mat-form-field class="flex-48">
+      <div formGroupName="clientNonPersonDetails" style="display: contents">
```

HTMLHint (1.9.2)

[warning] Inline style [  style="display: contents" ] cannot be used.

(inline-style-disabled)